src/recipe/dto/update-recipe-response.ts

```diff
@@ -0,0 +1,14 @@
+interface IUpdatedRecipe {
+  title: string;
+  description: string;
+  ingredients: string;
+  preparation: string;
+}
+
+export class UpdatedRecipeResponse {
+  constructor(public updatedRecipes: IUpdatedRecipe) {}
+
+  public static from(updatedRecipes: IUpdatedRecipe): UpdatedRecipeResponse {
```

Comment:
with this approach I think we will get a response like this:

```json
{
  updatedRecipes: {
    title: string;
    description: string;
    ingredients: string;
    preparation: string;
  }
}
```

I think this additional layer of nesting in this case is not needed, we should change it so that we get a result like this:

```json
{
  title: string;
  description: string;
  ingredients: string;
  preparation: string;
}
```